SimG4Core/Configuration/test/dd4hep_ttbar_2026D49_Step1_cfg.py

```diff
@@ -19,6 +19,10 @@
 process.load('Configuration.StandardSequences.EndOfProcess_cff')
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
 
+if hasattr(process,'MessageLogger'):
+    process.MessageLogger.categories.append('Geometry')
+    process.MessageLogger.categories.append('SimG4CoreApplication')
```

Comment:
As of next IB, this should *probably* be
```suggestion
    process.MessageLogger.cerr.Geometry = dict()
    process.MessageLogger.cerr.SimG4CoreApplication = dict()
```

I'm saying "probably" because the intention of only adding categories is not clear.

Reply:
> I'm saying "probably" because the intention of only adding categories is not clear.

The intention is here
https://github.com/cms-sw/cmssw/tree/master/FWCore/MessageService#have-specific-info-messages-be-displayed

Reply:
> > I'm saying "probably" because the intention of only adding categories is not clear.
> 
> The intention is here
> https://github.com/cms-sw/cmssw/tree/master/FWCore/MessageService#have-specific-info-messages-be-displayed

Right, but enabling `INFO` messages would need something else in addition to the categories, no?

Reply:
> Right, but enabling INFO messages would need something else in addition to the categories, no?

Not at the moment. The documentation was written in anticipation of us changing the default threshold. At the moment, we still use INFO as the threshold since changing it would break all the cases where just the categories are specified. It would take another pass to change those before we could consider changing the default threshold.

Reply:
Right, thanks. Then my suggestion in the first message should indeed be the way to go.